source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,310 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page you will learn the how to search **geospatial data** in the
+MongoDB Java Driver.
+
+Geospatial data is data that represents a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with **GeoJSON**.
+GeoJSON is a geospatial data format created by the Internet Engineering
+Task Force (IETF). You can use the GeoJSON format to store your data in MongoDB,
+and you must use the GeoJSON format to query your geospatial data in MongoDB. 
+
+For a concrete reference, GeoJSON data often resembles the following structure:
+
+.. code-block:: json
+
+   "<field name>" : {
+      "type": "<GeoJSON type>",
+      "coordinates": [
+         [<longitude>, <latitude>],
+         ...
+         [<longitude>, <latitude>]
+      ]
+   }
+
+For definitive information on GeoJSON, see the
+:rfc:`official IETF specification <7946>`.
+
+.. external resource
+
+GeoJSON Positions
+~~~~~~~~~~~~~~~~~
+
+A position is the base unit in GeoJSON, and represents a single place on Earth.
+A position is an array containing two or three number values: 
+
+- a longitude in the first position
+- a latitude in the second position
+- an optional elevation in the third position 
+
+.. tip:: Order Matters
```

Comment:
Consider a less general title so that the reader can identify whether they need to read the tip.
When reading this title, I think it's straightforward to interpret as "does the order of the coordinates in the array matter?" when it's actually the ordering of the latitude and longitude coordinate pair